### ISSUE TITLE
[one-cmds] analysis test failure with python3.8

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#8815]
With the draft upgrading to python3.8 and related packages, lots of one-cmds test fails.
let's find out the reason.

**Comment by seanshpark** [comment]
Refer https://github.com/Samsung/ONE/issues/8732#issuecomment-1090989108 and https://github.com/Samsung/ONE/issues/8732#issuecomment-1091014161

**Comment by seanshpark** [comment]
Done again clean build/test, twice
```
one-build_008.test FAILED
one-build_009.test FAILED
one-build_011.test FAILED
one-build_neg_006.test FAILED
one-import-onnx_001.test FAILED
one-import_005.test FAILED
one-import_006.test FAILED
onecc_008.test FAILED
onecc_009.test FAILED
onecc_011.test FAILED
onecc_016.test FAILED
onecc_neg_006.test FAILED
12 TESTS FAILED
```

**Comment by hseok-oh** [comment]
After docker image update, runtime test fails with below message

```
16:29:53 Traceback (most recent call last):
16:29:53   File "/jenkins_slave/workspace/nnfw/master/pr-x64-builder-tester/tests/nnapi/nnapi_test_generator/android-10/cts_generator.py", line 36, in <module>
16:29:53     import test_generator as tg
16:29:53   File "/jenkins_slave/workspace/nnfw/master/pr-x64-builder-tester/tests/nnapi/nnapi_test_generator/android-10/test_generator.py", line 39, in <module>
16:29:53     import numpy as np
16:29:53 ModuleNotFoundError: No module named 'numpy'
```

**Comment by seanshpark** [comment]
> ModuleNotFoundError: No module named 'numpy'

Yes, found out that for one-cmds test I need to add `tensorflow_probability` to `one-prepare-venv`

**Comment by seanshpark** [comment]
with adding `tensorflow_probability ` got 
```
ALL TESTS PASSED!
```

**Comment by hseok-oh** [comment]
I resolved my issue by installing python 3.6 pip and numpy on updated docker. 

```
$ python3 -m pip install --upgrade pip
$ python3 -m pip install numpy
```

We need to install numpy for both python3.6 and python3.8.
